apps/backoffice-v2/src/common/utils/export-to-csv/export-to-csv.unit.test.ts

```diff
@@ -143,12 +134,10 @@ describe('CSV Export Utils', () => {
       vi.clearAllMocks();
     });
 
-    test('should show error toast for empty data', () => {
+    test('returns false for empty data', () => {
```

Comment:
Consider keeping the previous format.
```suggestion
    test('should return false for empty data', () => {
```